src/cli/existing_mnemonic.rs

```diff
@@ -32,9 +32,9 @@ pub fn subcommand<'a, 'b>() -> App<'a, 'b> {
         .arg(
             Arg::with_name("chain")
                 .long("chain")
-                .required(true)
+                .required(false)
                 .takes_value(true)
-                .possible_values(&["goerli", "prater", "mainnet", "minimal", "holesky"])
+                .possible_values(&["goerli", "prater", "mainnet", "holesky"])
```

Comment:
why removing `minimal` and not adding `testnet`?

Reply:
`minimal` is not an actual preset, it was just a filler before to make suggestion for the library to use config file, so removing it actually fixes inconsistency here 
 For custom testnet, user needs to pass  `--testnet_config` parameter, pointing to yaml file, instead of  `--chain`, and omit `--chain` fully, so no sense to include `testnet` as well